include/circt/Dialect/FIRRTL/Passes.td

```diff
@@ -970,4 +970,16 @@ def ProbesToSignals : Pass<"firrtl-probes-to-signals", "firrtl::CircuitOp"> {
   let constructor = "circt::firrtl::createProbesToSignalsPass()";
 }
 
+def CheckUninferredResets : Pass<"firrtl-check-uninferred-resets", "firrtl::CircuitOp"> {
```

Comment:
This feels like it should be an interface pass so you can nest it on FModuleLike, rather than an op pass nested on CircuitOp.